cs/cli/vw_example.cpp

```diff
@@ -183,8 +183,8 @@ namespace VW
         vector<size_t> fa_missing;
         for (size_t ia = 0, ib = 0; ia < fa.values.size(); ia++)
         {
-            auto masked_weight_index = fa.indicies[ia] & vw->reg.weight_mask;
-            auto other_masked_weight_index = fb.indicies[ib] & vw->reg.weight_mask;
+            auto masked_weight_index = fa.indicies[ia] & vw->wv.mask();
```

Comment:
Again, perhaps `wv.index(fa.indicies[ia])`?  This is the last time I will mention this.